src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$main-font: "Roboto", helvetica, sans-serif;
```

Comment:
```suggestion
$main-font: "Roboto", Helvetica, sans-serif;
```

Reply:
src/styles/variables.scss
 1:23  ✖  Expected "Helvetica" to be "helvetica"   value-keyword-case 
(from npm run lint and npm test)